src/dms/tests/EdFi.DataManagementService.Tests.E2E/Features/References/KeyUnification.feature

```diff
@@ -16,9 +16,24 @@ Feature: Validation of Natural Key Unification
               And the system has these "Sessions"
                   | sessionName | schoolReference   | schoolYearTypeReference | beginDate  | endDate    | termDescriptor                                   | totalInstructionalDays |
                   | Session1    | {"schoolId": 123} | {"schoolYear": 2025}    | 2022-01-04 | 2022-05-27 | "uri://ed-fi.org/TermDescriptor#Spring Semester" | 88                     |
-              And the system has these "Courses"
-                  | courseCode | numberOfParts | identificationCode | schoolId | courseTitle  | educationOrganizationReference   | identificationCodes                                                                                                                                                                                                                                            |
-                  | Course1    | 1             | ALG-1              | 123      | Course1Title | {"educationOrganizationId": 123} | [{"courseIdentificationSystemDescriptor": "uri://ed-fi.org/courseIdentificationSystemDescriptor#LEA course code", "assigningOrganizationIdentificationCode": "IdentificationCode1", "courseCatalogURL": "URL12", "identificationCode": "IdentificationCode1"}] |
+              And a POST request is made to "/ed-fi/courses" with
```

Comment:
The old "Courses" insert above was not working - the code wasn't able to handle the `identificationCodes` properly. I gave up trying to figure out how to use the compact form (even though I generally prefer that form in a `Given` statement), and switched to using the JSON body below.